libraries/AP_HAL_ChibiOS/hwdef/DroneerF405/README.md

```diff
@@ -0,0 +1,95 @@
+# Droneer F405 Flight Controller
+
+The Droneer F405 is a flight controller produced by [Droneer](http://www.droneer.com/).
+
+## Features
+
+ - STM32F405 microcontroller
+ - MPU6000 IMU
+ - max7456 OSD
+ - 5 UARTs
+ - 9 PWM outputs
+
+## Pinout
+
+![Droneer F405](DroneerF405_Board_Bottom.jpg "Droneer F405")
+![Droneer F405](DroneerF405_Board_Top.jpg "Droneer F405")
+
+## UART Mapping
+
+The UARTs are marked Rn and Tn in the above pinouts. The Rn pin is the
+receive pin for UARTn. The Tn pin is the transmit pin for UARTn.
+
+ - SERIAL0 -> USB
+ - SERIAL1 -> UART1 (Tramp, DMA-enabled)
+ - SERIAL2 -> UART2 (RCIN, DMA-enabled) 
+ - SERIAL3 -> UART3 (ESC)
+ - SERIAL4 -> UART4 (GPS)
+ - SERIAL5 -> UART5 (None)
+
+## RC Input
+
```

Comment:
Replace the RC section with this:
```suggestion
The SBUS pin (SBU), is passed by an inverter to RX2 (UART2 RX), which can be used for all ArduPilot supported receiver protocols, except CRSF/ELRS and SRXL2. CRSF/ELRS and SRXL2 require the RX2 and TX2 pin be connected instead of the SBUS pin.

* SRXL2 requires SERIAL2_OPTIONS be set to “4”.
* Fport should be connected to TX2 via a bi-directional inverter and SERIAL2_OPTIONS be set to “4”.

Any UART can also be used for RC system connections in ArduPilot and is compatible with all protocols except PPM (SBUS requires external inversion on other UARTs). See Radio Control Systems for details.
```